internal/controllers/topology/cluster/reconcile_state.go

```diff
@@ -727,3 +727,24 @@ func (r *Reconciler) reconcileReferencedTemplate(ctx context.Context, in reconci
 
 	return nil
 }
+
+// CreateError removes details about the specific name of the object from the Error message. The full names include a hash
```

Comment:
Can you please add unit tests including 1-2 cases with real StatusErrors? I have a hard time figuring out how the final error will look like

Reply:
I've added a test now to show what the output should be.

Reply:
Thx!